### PR TITLE
Query multiple table

### PR DESCRIPTION
**Pull request by leloufadel**
in this pull request: 
I ensured the General requirements bellow. 
```
> **General requirements**
> Make sure that you used correct ([Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md)).
> Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
```
**Project requirements**

- I Created a table named owners with the following columns:
- id: integer (set it as autoincremented PRIMARY KEY)
- full_name: string
- age: integer
- I Created a table named species with the following columns:
- id: integer (set it as autoincremented PRIMARY KEY)
- name: string

** I Modified animals table:**

- Ensured that id is set as autoincremented PRIMARY KEY
- Removed column species.
- Added column species_id which is a foreign key referencing species table
- Added column owner_id which is a foreign key referencing the owners table
- all this goes in schema.sql file.

**Inserted the following data into the owners table:**

> Sam Smith 34 years old.
> Jennifer Orwell 19 years old.
> Bob 45 years old.
> Melody Pond 77 years old.
> Dean Winchester 14 years old.
> Jodie Whittaker 38 years old.
Capture-15A

Insert the following data into the species table:
Pokemon
Digimon
Capture-17-A

**Modified inserted animals so it includes the species_id value:**
If the name ends in "mon" it will be Digimon
Capture-18A

All other animals are Pokemon
Capture19A

**Modified inserted animals to include owner information (owner_id):**
Sam Smith owns Agumon.
Jennifer Orwell owns Gabumon and Pikachu.
Bob owns Devimon and Plantmon.
Melody Pond owns Charmander, Squirtle, and Blossom.
Dean Winchester owns Angemon and Boarmon.
Capture19A-
these insertions and modifications happen in data.sql file.

**Writed queries (using JOIN) to answer the following questions:**
1 . What animals belong to Melody Pond?

Capture20B-Reviews

2 . List of all animals that are pokemon (their type is Pokemon).

Capture21B

3 . List all owners and their animals, remember to include those that don't own any animal.
Capture22B

4 . How many animals are there per species?

Capture23B

5 . List all Digimon owned by Jennifer Orwell.

Capture24B

6 . List all animals owned by Dean Winchester that haven't tried to escape.

Capture21B-Reviewers

7 . Who owns the most animals?

Capture26B

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -96,6 +96,52 @@ Select species, AVG(escape_attempts)
...
-
+-----Write queries (using JOIN) to answer the following questions:
+---What animals belong to Melody Pond?
+
+select animals.name, owners.full_name
+from animals 
+inner join owners on 
+animals.owner_id = owners.id
+where owners.full_name ='Melody Pond'
+;
```
Comment (danmainah) [inline queries.sql:107]: - [ ] @leloufadel Great work adding a query to find what animals belong to Melody Pond.However the screenshot provided doesn't showcase the same query and expected result kindly consider changing it.

File: queries.sql [hunk 2]
```diff
@@ -96,6 +96,52 @@ Select species, AVG(escape_attempts)
...
+inner join owners on owners.id = animals.owner_id
+where owners.full_name = 'Jennifer Orwell' and species.name ='Digimon'
+;
+--List all animals owned by Dean Winchester that haven't tried to escape.
+
+select animals.name, owners.full_name, animals.escape_attempts
+from animals
+inner join owners on owners.id = animals.owner_id
+where owners.full_name = 'Dean Winchester' and animals.escape_attempts =0
+;
```
Comment (danmainah) [inline queries.sql:139]: - [ ] @leloufadel Great work adding a query to check all animals owned by Dean Winchester that haven't tried to escape, however, the screenshot doesn't show the query result, kindly consider showcasing the query result of this query.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @leloufadel , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Added the data given correctly
✔️  Added the new tables with the correct columns and datatypes.


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- In your getting started section you could add that the user can run the data , queries and schema files to check how the project works.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by LionRouge1 — APPROVED** [review]
Hi @leloufadel, 

# STATUS: APPROVED! 🥇 💯 
Great work on making the changes 👏 
You've done well implementing the requested changes

done

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

## Highlights
- Add screenshots of the queries result to the PR :heavy_check_mark:  
- Nice work you meet all the requirements :1st_place_medal: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**